File: Programming-Basics-with-Python/9.While Loop - Lab/05. Account Balance.py
# команда -> сума за вноска (под формата на текст) или  "NoMoreMoney"

# повтаряме: четем сума + проверка < 0
# спираме: command == "NoMoreMoney"
# продължаваме: command != "NoMoreMoney"
command = input()
total_money = 0 # сума на сметката
while command != "NoMoreMoney":
    # command -> сума за вноска '5.51'
    sum_income = float(command)
    if sum_income < 0:
        print('Invalid operation!')
        break
    else:
        print(f"Increase: {sum_income:.2f}")
        total_money += sum_income
    command = input()

print(f"Total: {total_money:.2f}")

File: Programming-Basics-with-JavaScript/5.While Loop/05accountBalance.js
function accountBalance(input){
    let index = 0
    let account = 0

    let command = input[index]
    index++

    while (command !== 'NoMoreMoney'){
        money = Number(command)

        if (money < 0){
            console.log("Invalid operation!")
            break
        }
        account += money
        console.log(`Increase: ${money.toFixed(2)}`)

        command = input[index]
        index++
    }
    console.log(`Total: ${account.toFixed(2)}`)
}
accountBalance(["120",
"45.55",
"-150"])

